mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerClient.java

```diff
@@ -316,10 +317,10 @@ public DataManagerClient(BaseApiManager baseApiManager,
      *
      * @param clientId          Client Id
      * @param identifierPayload IdentifierPayload
-     * @return GenericResponse
+     * @return IdentifierCreationResponse
      */
-    public Observable<GenericResponse> createClientIdentifier(int clientId,
-                                                              IdentifierPayload identifierPayload) {
+    public Observable<IdentifierCreationResponse> createClientIdentifier(
```

Comment:
@therajanmaurya We are not fetching the identifiers list again. We are retrieving the created identifier's id from the response based on the discussion with @droidchef  That's why I changed it.